### DISCUSSION TITLE
Z13 GZ301VV lose gpu fan speed control

### DISCUSSION THREAD

**Discussion by rainbowflesh** [seerge/g-helper#1399] [Q&A]
Not totally lose control, but GPU fan control running into a wired statue:
1. fan won't stop after temperature cooled down, and keep running in minimal speed or more, sometimes in 40% speed.
2. fan have burst run (0-50-0 percent within seconds) when some software calling dGPU.
3. Adjust custom GPU fan curves to all in 0% or 1% will let fan keep running in 50% speed, wtf.
4. Adjust custom GPU fan curves to very high fan starting temperature will make fan stop in low temp *a while*, like few minutes, and back to high speed.

Something that can be confirmed:
- not hardware issue, fan can keep in stable speed.
- not sensor issue, fan still adjusting speed when temp change.
- not my issue, I just notice this today due to the unexpected fan noise.
- Only `silent` mode can turn off dGPU fan, in `balance` and `turbo`, change GPU mode or power mode won't help.
- I tested some old version ghelper, The issues still there, perhaps not ghelper's issues.
- Didn't use AC with G-Helper.

Something that IDK what's going on:
- bios update, there recently have a performance boots upgrade.

------

![4](https://github.com/seerge/g-helper/assets/20819692/f15187c9-6cb0-4cb9-9cd1-74bc7fc6be4f)
![3](https://github.com/seerge/g-helper/assets/20819692/415b27b9-6c1f-4576-83b4-ed3b172f05af)
![2](https://github.com/seerge/g-helper/assets/20819692/e3e44da7-b152-4872-8563-9d657e45e840)

**Comment by TehH4rRy** [comment]
I'm getting similar issues with the GPU fan curves. I have mine set to sit at 2000rpm until it gets over 60 degrees, but at 44 it's jumping constantly between 2000 and 2300rpm. So annoying to listen to. 
![image](https://github.com/seerge/g-helper/assets/987285/b65e57be-fda1-40c1-8b65-25588092911c)

**Reply by rainbowflesh** [reply]
Adjust fan curves to 55℃ off for all profile, and change between `silent` `balance` `turbo` see if GPU fan could be stopped; Mine just go 4000 rpm minimal fan speed in turbo, 2300 rpm in balance but stopped at `slient`, I think there would be a bios issue.
